app/views/search_results/_search_result.html.erb

```diff
@@ -0,0 +1,9 @@
+<% present(search_result, SearchResultPresenter) do |search_result| %>
```

Comment:
I think after the https://github.com/moneyadviceservice/site_search/pull/2 you can remove the present call.